### ISSUE TITLE
Spleeter won't produce the output in .mp3 in Windows 

### ISSUE THREAD

**Issue by MohammedMehdiTBER** [deezer/spleeter#663]
- [ ]  I didn't find a similar issue already open.

- [ ]  I read the documentation (README AND Wiki)

- [ ] I have installed FFMpeg

- [ ]  My problem is related to Spleeter only, not a derivative product (such as Webapplication, or GUI provided by others)

## Description

Spleeter won't produce the output in `.mp3` in Windows, but It does on Debian Linux. Still, It works for `.flac`and `.wav` file formats.

## Step to reproduce

1. Installed dependencies of `ffmpeg` using `conda` and run `conda update --all`  and Installed Spleeter using updated  `pip` and python 3.7/8/9 (tested on different versions) 
2. Using 30 GB of virtual memory, I run in `conda prompt` as ` spleeter separate --verbose -p spleeter:2stems-16kHz -f {filename}-{instrument}.{codec} -d "7200" -c "mp3" -b "320k" --mwf -B "tensorflow" -o "output_mwf" "C:\Users\MohammedMehdiTBER\Music\example.mp3" `
3. Got nothing after minutes of processing.

## Output

```DOS
(base) C:\Users\MohammedMehdiTBER\Music\Spleeter> spleeter separate --verbose -p spleeter:2stems-16kHz -f {filename}-{instrument}.{codec} -d "7200" -c "mp3" -b "320k" --mwf -B "tensorflow" -o "output_mwf" "C:\Users\MohammedMehdiTBER\Music\example.mp3"

(base) C:\Users\MohammedMehdiTBER\Music\Spleeter>
```

## Environment

| OS                | Window |
| Installation type | Conda / pip |
| RAM available     | 6G Physical 30GB Virtual                             |
| Hardware spec     | Intel Core i5 gen 2            |

**Comment by romi1502** [comment]
Hi @MohammedMehdiTBER,
I cannot reproduce on Linux. The output files are properly written after a few seconds.
Can you provide more details. Particularly:
- I see you use the `-d 7200` option. Are the file you're processing very long? have you tried with shorter files?
- Do you perform the separation on CPU or GPU?

**Comment by MohammedMehdiTBER** [comment]
No, that duration is just long actually for making sure to process all the files that I don't know their duration in seconds. I don't have CUDA, so I may be using just CPU, and It makes the device frozen during the process until It finishes.

**Comment by romi1502** [comment]
Ok. I see that you have a "zsh: killed" log. Did you kill the process or did it die by itself?
